manifests/config/exporter/exporter.yaml

```diff
@@ -54,6 +54,7 @@ spec:
         key: node-role.kubernetes.io/master
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccountName: kepler-sa
+      hostPID: true
```

Comment:
This is important, otherwise we cannot get the right container ID and then pod name.